numba/tests/test_np_functions.py

```diff
@@ -3621,6 +3626,71 @@ def test_cross2d_exceptions(self):
             str(raises.exception)
         )
 
+    def test_tile(self):
+        def tiles():
+            yield 2
+            yield 3.0
+            yield True
+            yield np.array([])
+            yield np.array([1, 2])
+            # yield np.array([[1, 7], [3, 8]], dtype=float) - not supported yet
+
+        def shapes():
+            yield 10
+            yield 0
+            yield (6,)
+            yield (3, 3)
+            yield (2, 1, 4)
+
+        pyfunc = tile
+        cfunc = njit(pyfunc)
+        for a, reps in product(tiles(), shapes()):
+            print(a, reps)
+            expected = pyfunc(a, reps)
+            got = cfunc(a, reps)
+            self.assertPreciseEqual(expected, got)
+
+    def test_tile_exceptions(self):
+        pyfunc = tile
+        cfunc = njit(pyfunc)
```

Comment:
Is the naming `cfunc` for jitted functions in the tests a convention throughout Numba? Seems a little misleading in my opinion.

Reply:
Ok I see it is a convention. I guess it stands for "compiled" func or something. It is ok then.